lib/Arch/AArch32/ArchContext.cpp

```diff
@@ -0,0 +1,10 @@
+#include <remill/Arch/AArch32/ArchContext.h>
+
+namespace remill {
+const std::string_view kThumbModeRegName = "TMReg";
```

Comment:
Any special reason why these are not declared in header? Because it will improve both readability and optimisations.